docs/about/README.md

```diff
@@ -54,16 +59,68 @@ Depending on the mode used, the main domain (e.g. `www.domain.com`) will be conn
+
+![](https://w3techs.com/diagram/history_technology/cm-wordpress)
+
+With the shift to Gutenberg, and as the use of [headless CMS](https://css-tricks.com/what-is-a-headless-cms/) grows, the WordPress community has increasingly started considering React for their projects. Besides this, modern libraries like React are growing in popularity and becoming essential to rich user experiences.
+
+If **WordPress** is great and **React** is too, then why not **combine** the two? Especially if you want to build a CMS-powered site with modern web development tools.
+
+We believe this JavaScript-based approach is gaining traction in the WordPress ecosystem, so there’s no better time to start getting familiar with it.
+
+
+## Why Frontity?
+
+Going with Frontity in the stack React + WordPress means you get the best of WordPress and the best of React while the framework does the heavy lifting. Therea are several important benefits/reasons for using Frontity.
+
+### Benefits of using React and WordPress with Frontity
+
+#### The Developer Experience
+
+Frontity makes your life easier through a zero setup frameworkf and several tools and features, that typically require custom engineering, available out of the box.
+
+#### The Extensibility
+
+Frontity framework helps minimize both development and maintenance times. The extensibility through packages and the ability to reuse them helps reduces the time and effort necessary to create and maintain a project.
+
+
+#### Easy to get started with
+
+There is a small learning curve but even developers new to React can get up and running relatively quickly.
+
+
+#### Strong performance-oriented approach
+
+Frontity’s infrastructure is designed to be as efficient and performant as possible. Strategies directly available with Frontity such as WordPress data normalization, lazy loading, prefetching or code splitting will have a positive impact on the performance and UX of your projects
+
+#### SEO optimized
+
+Frontity is Search Engine Optimized. The SSR approach will generate HTML from your React code making your pages perfectly indexable bt SEO engines. With Frontity you can have complete SEO control through some Frontity packages that will add support for WordPress SEO plugins.
+
+#### The Editing Experience
+
+With WordPress the editing experience is great. Marketing teams can benefit from real-time content updates, preview support with Embedded Mode and much more.
+
+#### The User Experience
+
+Users will get meaningful content as soon as possible. Then, the Hydration process will make React to take control of the App so following pages will load instantaneously because of pre-fetching and some other strategies integrated with Frontity. Having pages loading instantaneously help to create a great User Experience
```

Comment:
~~Users will get meaningful content as soon as possible. Then, the Hydration process will make React to take control of the App so following pages will load instantaneously because of pre-fetching and some other strategies integrated with Frontity. Having pages loading instantaneously help to create a great User Experience~~

Users get meaningful content as quickly as possible. The Hydration process enables React to take control of the App so that subsequent pages load instantaneously. This is due to pre-fetching and other strategies employed by Frontity. Having pages load instantaneously helps to create a great User Experience.